src/os/bluestore/BlueStore.cc

```diff
@@ -5257,6 +5257,12 @@ int BlueStore::_setup_block_symlink_or_file(
   if (create)
     flags |= O_CREAT;
   if (epath.length()) {
+    if (!create) {
```

Comment:
I think this needs to go down under the SPDK case below; in that case the symlink points to an PCI device id that isn't an actual file.